login Page: Email auto-complete and password auto focus

thrishank opened the pull request:
### PR Fixes: #843
- email domain suggestion as user enters the email
- password auto focus on entering the email
- This PR gets rid of using mouse when signing which significantly improves the UX
### Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I assure there is no similar/duplicate pull request regarding same issue

hkirat commented:
<img width="943" alt="Screenshot 2024-08-04 at 6 46 12 AM" src="https://github.com/user-attachments/assets/2a18b05c-2129-4e33-9faa-022f7cfcdb6a">
Rather try something like this -

thrishank commented:
done sir @hkirat 
1. Added email domain suggestions
2.  Added keyboard functionality to access this domain suggestions
3. Make sured that password is always in focus after entering the email
[screen-capture.webm](https://github.com/user-attachments/assets/277c8bd7-b57e-4491-8c8c-d327d53b8186)

thrishank commented:
hey @devsargam this PR is similar to #848 can you review it once
